distribution/docs/src/main/jdocs/content/_plugins/tags-filter-plugin-contents.adoc

```diff
@@ -0,0 +1,23 @@
+:type: plugin
+:status: published
+:title: Tags Filter Plugin
+:link: _tags_filter_plugin
+:plugintypes: prefederatedquery
+:summary: Updates queries without filters.
+
+The Tags Filter Plugin updates queries without filters for tags, and adds a default tag of `resource`.
+For backwards compatibility, a filter will also be added to include metacards without any tags attribute.
```

Comment:
I agree. I think there is a difference, but I will confirm and update under DDF-3160.